src/commands/stats.ts

```diff
@@ -258,20 +251,11 @@ async function leaderboard(
 		throw new UserMessageError(`No one is tracking the stat "${statName}"`);
 	}
 
-	const scoresWithUsernames = scoreboardEntries
-		.map(entry => ({
-			username: userCache.get(entry.userId)?.username,
-			userId: entry.userId,
-			score: entry.score,
-		}))
-		.sort((a, b) => b.score - a.score);
+	const scoresSorted = scoreboardEntries.sort((a, b) => b.score - a.score);
 
-	const embedDescription = scoresWithUsernames
+	const embedDescription = scoresSorted
 		.map(entry => {
-			if (entry.username === undefined) {
-				throw new Error(`Unable to find user with id ${entry.userId}`);
-			}
-			return `${entry.username}: ${entry.score}`;
+			return `<@${entry.userId}>: ${entry.score}`;
```

Comment:
It might be better to use d.js's built-in [user mention](https://discord.js.org/#/docs/discord.js/main/function/userMention) formatter instead of doing it ourselves.